tools/restarts_to_diags.py

```
@@ -0,0 +1,64 @@
+    try:
+        from pyphare.simulator.simulator import startMPI
+
+        startMPI()
+        main()
+    except Exception as e:
+        import traceback
+
+        print(f"Exception caught in phare/tools/restarts_to_diags.py: \n{e}")
+        print(traceback.format_exc())
+        sys.exit(1)
+    except ...:
```

Ruff (0.14.5)

`except` handlers should only be exception classes or tuples of exception classes

(B030)